kubric/simulator.py

```diff
@@ -11,48 +11,161 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
I'm going to skip `simulator.py` as you own that part.
The logic is the same as `blender.py` I assume for when it comes to the use of traitlets?

Reply:
yes. Except that I had to write specialised setters for the individual attributes, because of the way pybullet groups properties together.